### ISSUE TITLE
(With Iris/Indium/Starlight) - Game crashes when player takes damage from mobs

### ISSUE THREAD

**Issue by SnowyOwlNugget** [Charles445/DamageTilt#21]
When taking damage from a cave spider, the game will stop moving, wait a bit, then crash. Removing Damage tilt did fix the issue, so I'm at least moderately sure it's a damage tilt issue.
EDIT: iirc the majority of 1.19.4 was performance updates, right? so maybe it's a ticking error because it's the wrong version (from my little knowledge the logs seem to hint at this), but I'm just surprised that if it's such a game-breaking error it wouldn't crash on launch.

On fabric loader 0.14.19
Mod List-
Damagetilt 0.1.2
Continuity 3.0 beta2
DistantHorizons 1.6.11a
FabricAPI 0.79.0
Indium 1.0.15
Iris 1.6.1
Journeymap 5.9.5
Lithium-fabric 0.11.1
modmenu 6.2.1
smooth swapping 0.8.1
sodium-fabric 0.4.10 build24
starlight 1.1.1

Log:
[dtilitcrashlog.txt](https://github.com/Charles445/DamageTilt/files/11334529/dtilitcrashlog.txt)
Vid:
https://user-images.githubusercontent.com/49455179/234623926-b5c0316d-7439-4d25-b256-5168f75e6e88.mp4

**Comment by Charles445** [comment]
If I'm not mistaken, damage tilt is re-added to 1.19.4 on its own, so you don't need the mod anymore!
Try removing the mod and seeing if damage is tilting your camera in the direction like it should

**Comment by SnowyOwlNugget** [comment]
Ah, i'll try it right now, give me a moment.
Edit: No, it seems to only move left and right in 1.19.4 for me. Same Mods.

**Comment by Charles445** [comment]
That's strange, I just tested those 11 other mods on my end by themselves on 1.19.4, and it seems to be working.

Try spawning in a skeleton and having it hit you with an arrow, they're the best for testing it.

Also check the accessibility setting 'Damage Tilt', make sure it's 100%

If that still doesn't work then maybe something weird happened with the 1.19.4 install?

**Comment by SnowyOwlNugget** [comment]
yeah, the install might be borked. I orignally had this issue on my laptop, and when I tried to transfer the mods and their settings over to a desktop, the quilt launcher just kept saying there was like no java executable that it could find, even when I literally put in in the folder with java lol. Eventually it simply decided it could find a javaw.exe it liked, because it began working,(Just copying the quilt-whateverlauncher.jar version always errored out, so it seems like it actually did work) but I hadn't readded damagetilt to that list as it broke other things in the past. I'll try this on my desktop tomorrow. Maybe on my laptop, through some stroke of luck it manages to launch, but might still be broken somehow. (I'll DEFINITELY make a copy cuz I don't want to deal with the installer bs again tho lmao) TLDR will attempt to verify game files, but quilt is weird lol.

**Comment by SnowyOwlNugget** [comment]
Actually, If you want to try to recreate my issue, I've happened to already upload the directory to google drive, so I'll just post it here.
Game DIR: https://drive.google.com/drive/folders/1VwpVkYdgvTk73pqwX_QhnKAtIz31_G3z?usp=sharing

DIR JVM ARGS :https://docs.google.com/document/d/175DQIjlSBfdoET1nLN83528O1riQLhOMGsrfC6dlFh0/edit?usp=sharing

**Comment by xzTWINzx** [comment]
same problem, how fix it?
